frontend/Pages/ProfileConfigPage/index.tsx

```diff
@@ -243,17 +243,19 @@ export const ProfileConfigPage: React.FC = () => {
           <Card.Content>
             <View style={styles.cardPicture}>
               <TouchableRipple onPress={() => bottomSheetPictureRef.current?.open()}>
-                {user?.picture ? (
-                  <Avatar.Image size={100} source={{ uri: user.picture }} />
-                ) : (
-                  <NostrosAvatar
-                    name={user?.name}
-                    pubKey={nPub ?? ''}
-                    src={user?.picture}
-                    lud06={user?.lnurl}
-                    size={100}
-                  />
-                )}
+                <View style={{borderRadius: 50, overflow: 'hidden'}}>
+                  {user?.picture ? (
+                      <Avatar.Image size={100} source={{ uri: user.picture }} />
```

Comment:
I have no idea why this looks like this, but I don't think this line makes sense, I believe you can just remove it and leave `NostrosAvatar`